LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2021 Queen’s Printer for Ontario [ [MosCD3](https://github.com/MosCD3) - Mostafa Youssef, [nbAmit](https://github.com/nbAmit) - AMit Padmani, [NB-PrasadKatkar](https://github.com/NB-PrasadKatkar) - Prasad Katkar, [NB-MikeRichardson](https://github.com/NB-MikeRichardson) - Mike Richardson]
```

Comment:
Licenses are not printed as markdown, so I think it would be cleaner to loose the markdown formatting. Also could we add a generic hyperledger contributors clause?

```suggestion
   Copyright 2020-present Hyperledger Contributors.
   Copyright 2021 Queen’s Printer for Ontario. Mostafa Youssef (https://github.com/MosCD3), Amit Padmani (https://github.com/nbAmit), Prasad Katkar (https://github.com/NB-PrasadKatkar), Mike Richardson (https://github.com/NB-MikeRichardson)
```